spark/src/main/scala/io/substrait/spark/expression/FunctionMappings.scala

```diff
@@ -77,7 +97,8 @@ class FunctionMappings {
     s[Min]("min"),
     s[Max]("max"),
     s[First]("any_value"),
-    s[HyperLogLogPlusPlus]("approx_count_distinct")
+    s[HyperLogLogPlusPlus]("approx_count_distinct"),
```

Comment:
tehcnically the "approx_count_distinct" says it's HyperLogLog while this is HLL++, but given the ++ should just be better that's probably fine!